### ISSUE TITLE
AnvilGUI does not support server version "1_20_R3"

### ISSUE THREAD

**Issue by SrBedrock** [TheRedEnd2000/AdvancedEggHunt#2]
```
[06:52:28 ERROR]: Could not pass event InventoryClickEvent to AdvancedEggHunt v3.0.0
java.lang.ExceptionInInitializerError: null
        at de.theredend2000.dependency.anvilgui.AnvilGUI$Builder.<init>(AnvilGUI.java:336) ~[AdvancedEggHunt v3.0.0.jar:?]
        at de.theredend2000.advancedegghunt.managers.inventorymanager.collection.CollectionCreator.handleMenu(CollectionCreator.java:67) ~[AdvancedEggHunt v3.0.0.jar:?]
        at de.theredend2000.advancedegghunt.listeners.InventoryClickEventListener.onClickInventory(InventoryClickEventListener.java:39) ~[AdvancedEggHunt v3.0.0.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor259.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[purpur-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:77) ~[purpur-api-1.20.4-R0.1-SNAPSHOT.jar:git-Purpur-2159]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:617) ~[purpur-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleContainerClick(ServerGamePacketListenerImpl.java:3180) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:58) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundContainerClickPacket.handle(ServerboundContainerClickPacket.java:23) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:54) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1498) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:194) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1475) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1398) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1376) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1265) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323) ~[purpur-1.20.4.jar:git-Purpur-2159]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalStateException: AnvilGUI does not support server version "1_20_R3"
        at de.theredend2000.dependency.anvilgui.version.VersionMatcher.match(VersionMatcher.java:32) ~[AdvancedEggHunt v3.0.0.jar:?]
        at de.theredend2000.dependency.anvilgui.AnvilGUI.<clinit>(AnvilGUI.java:43) ~[AdvancedEggHunt v3.0.0.jar:?]
        ... 27 more
Caused by: java.lang.ClassNotFoundException: de.theredend2000.dependency.anvilgui.version.Wrapper1_20_R3
        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:199) ~[purpur-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:166) ~[purpur-api-1.20.4-R0.1-SNAPSHOT.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
        at java.lang.Class.forName0(Native Method) ~[?:?]
        at java.lang.Class.forName(Class.java:375) ~[?:?]
        at de.theredend2000.dependency.anvilgui.version.VersionMatcher.match(VersionMatcher.java:28) ~[AdvancedEggHunt v3.0.0.jar:?]
        at de.theredend2000.dependency.anvilgui.AnvilGUI.<clinit>(AnvilGUI.java:43) ~[AdvancedEggHunt v3.0.0.jar:?]
        ... 27 more
```

**Comment by SrBedrock** [comment]
With #3 

![image](https://github.com/TheRedEnd2000/AdvancedEggHunt/assets/51332006/78ab55a3-357e-4535-bcc8-c204337b4429)

**Comment by TheRedEnd2000** [comment]
Fixed in 3.0.1
